### PR TITLE
Dropping Python 2: Removed Python 2 compatibility

### PR DESCRIPTION
**Pull request by iamabhishek0**
#### Brief description of what is fixed or changed
Removed code that was Python 2 compatible

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/core/tests/test_expr.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 # replace 3 instances with int when PY2 is dropped and
 # delete this line
-_rint = int if PY3 else float
+_rint = int
```
Comment (oscarbenjamin) [inline sympy/core/tests/test_expr.py]: I think it would be better to delete this and use `int` instead of `_rint`

**Comment by oscarbenjamin** [comment]
I left one comment but otherwise this PR looks good

**Comment by iamabhishek0** [comment]
@oscarbenjamin I have updated the PR with the suggested change.

**Comment by oscarbenjamin** [comment]
This looks good. Thanks for this!

**Pull request merged**